README.md

```diff
@@ -10,18 +10,17 @@ Obtain an access TOKEN from [stattleship.com](www.stattleship.com). Load R and i
 
 ```
 library(stattleshipR)
-TOKEN <- 'insert-your-token-here'
+setToken('insert-your-token-here')
 ```
 
 Get all NBA players:
 
 ```
-library(stattleshipR)
 league = "nba"
 sport = "basketball"
 ep = "players"
 q_body = list()
-players = stattle(TOKEN, sport=sport, league=league, ep=ep, query=q_body, version=1, verbose=TRUE, walk=TRUE)
+players = stattle(sport=sport, league=league, ep=ep, query=q_body, version=1, verbose=TRUE, walk=TRUE)
 players_df = do.call("rbind", lapply(players, function(x) x$players))
```

Comment:
We aren't hardcoding this in yet because combining all of the results into one `data.frame` breaks on some API calls e.g. feats. So we need to figure out all of the cases first so we properly manipulate the list structure into a `data.frame` for all calls. Would rather get the user a result than have it break and not allow them to get any data back. Or we could add a `try` type of behavior that returns the `data.frame` only if the combining works with no warning or error until we fix the edge cases.